share/spice/dota2/dota2.js

```diff
@@ -0,0 +1,148 @@
+(function (env) {
+    "use strict";
+
+    function getItemFromData(itemdata, itemName) {
+        var item;
+        if (itemdata[itemName]) {
+            item = itemdata[itemName];
+        } else {
+            $.each(itemdata, function(index, value) {
+                if (value.dname.toLowerCase() == itemName) {
+                    item = value;
+                }
+            });
+        }
+        return item;
+    }
+
+    function getItemComponents(item, fullResult) {
+        var components = [];
+        if (item.components) {
+            $.each(item.components, function(index, value) {
+                var component = {
+                    'name': fullResult.itemdata[value].dname,
+                    'img': 'http://cdn.dota2.com/apps/dota2/images/items/' + fullResult.itemdata[value].img,
```

Comment:
You'll want to use our DDG proxy function here for the images: https://github.com/duckduckgo/duckduckgo-utils/blob/f1a90299a4a7330681902e91f59dc650f791fb4c/util.js#L255